### PR TITLE
Completes ION-1036 which adds support for uploading SBOM files.

### PR DESCRIPTION
**Pull request by juwilliams**
# Updates
- added support for uploading SBOMs to Yolanda via anion-sdk.

[ION-1036] added support for importSBOM to projects lib.

[ION-1036] added append auth headers call to importSBOM.

[ION-1036] unwrapping response from importSBOM.

[ION-1036] added function to fetch and return the result of uploading an sbom via getSBOM endpoint.

[ION-1036] added confirmSbomEntry support.

[ION-1036]: https://ionchannel.atlassian.net/browse/ION-1036
[ION-1036]: https://ionchannel.atlassian.net/browse/ION-1036
[ION-1036]: https://ionchannel.atlassian.net/browse/ION-1036
[ION-1036]: https://ionchannel.atlassian.net/browse/ION-1036
[ION-1036]: https://ionchannel.atlassian.net/browse/ION-1036

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "anion-sdk",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (matthewkmayer) [inline package.json:3]: Since `master` is already at 1.2.1, I think we want 1.2.2 here.

**Review by ianling — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ianling) [reply package.json:3]: Weird that it shows as a change.... I thought Github removed stuff like that from the change list if it was already present

File: lib/projects/projects.js [hunk 2]
```diff
@@ -98,13 +98,69 @@ export class Projects {
...
+        throw error
+      })
+  }
+
+  static getImportSbomResult({ id }) {
+    const params = new URLSearchParams({
+      id,
+    })
+
+    return Post(getSbomUploadResultEndpoint, params, Auth.appendHeaders())
```
Comment (matthewkmayer) [inline lib/projects/projects.js:131]: This one's on me: the endpoint happens to respond to POST but is meant to only handle GET. Can we switch this  to use GET instead?

File: package.json (hunk 1 above)
Comment (juwilliams) [reply package.json]: done

File: lib/projects/projects.js (hunk 2 above)
Comment (juwilliams) [reply lib/projects/projects.js]: done

**Review by matthewkmayer — APPROVED** [review]

**Pull request merged**